src/person.rb

```diff
@@ -2,14 +2,16 @@
 require_relative './decorator'
 
 class Person < Nameable
-  attr_accessor :name, :age, :parent_permission
+  attr_accessor :name, :age, :parent_permission, :rentals
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
+    @id = Random.rand(1...100_000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Please create a `add_rental` method which accepts a `date` and a `book` so that a `Person` can successfully add a rental 👍